packages/amplify_flutter/lib/config/auth/login_mechanisms.dart

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+enum LoginMechanisms {
+  EMAIL,
```

Comment:
Prefer following Dart conventions for [enums](https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names) (camel case). Can adjust JSON value as I mentioned

```dart
enum LoginMechanisms {
  @JsonValue('EMAIL')
  email,

  // etc.
}
```

Reply:
Done!